tests/nb/_fixtures.yaml

```
@@ -67,9 +81,13 @@ entities:
   - name: "Stuelampe"
     id: "light.living_room_lamp"
     area: "living_room"
-    state: "on"
+    state:
+      in: "på"
+      out: "on"
 
   - name: "Garasjelys"
     id: "light.garage"
     area: "garage"
-    state: "on"
+    state:
+      in: "på"
+      out: "on"
```

Just add an empty line in the end of the file
```suggestion
      out: "on"

```

---

Thank you. That was an easy fix.